tests/bluetooth/tester/src/l2cap.c

```diff
@@ -58,6 +59,17 @@ static int recv_cb(struct bt_l2cap_chan *l2cap_chan, struct net_buf *buf)
 	tester_send(BTP_SERVICE_ID_L2CAP, L2CAP_EV_DATA_RECEIVED,
 		    CONTROLLER_INDEX, recv_cb_buf, sizeof(*ev) + buf->len);
 
+	/* TODO should be list but since currently only 1 credit is
+	 * supported we can keep it simpler for now
+	 */
+	if (!chan->nbuf) {
+		/* no need for extra ref, as when returning EINPROGRESS user
```

Comment:
I would avoid the word "user" here. Can we say "This function gains ownership of reference `buf` if it returns -EINPROGRESS.".